### ISSUE TITLE
Matrix: Sc:KTIq1 electron transfer activity (tRNA metabolism/energy generation)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2446]
KTI11	Zn-ribbon protein that co-purifies with Dph1 and Dph2		electron transfer activity		SGD	Saccharomyces cerevisiae S288C	IDA		family not named pthr21454	gene		PMID:18021800	20080102

this term has a parent to electron transport chain and is defined as
Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient.

which seems unlikely that this is the functional role based on the other annotations and location (maybe moving up to oxidoreductase would be better?)

**Comment by srengel** [comment]
Hi @ValWood this paper is still in our curation queue for GO and indicates KTI11 is an electron carrier via its bound iron for DPH1-DPH2:
https://www.ncbi.nlm.nih.gov/pubmed/24422557

these papers believe them: 
https://www.ncbi.nlm.nih.gov/pubmed/25543256
https://www.ncbi.nlm.nih.gov/pubmed/25604895

seems legit to me.  i'm inclined to leave it as is, and add annotations from these other papers as well.

what do you think?

**Comment by ValWood** [comment]
Hmm, I think you can be an "electron donor" without being part of the respiratory chain.

I  remember a ticket about this once  (possibly multiple) and in these cases we just use "oxidoreductase" 

"Electron carrier" == "electron transporter"  is a bit different

GO:0009055    electron transfer activity

Definition (GO:0009055 GONUTS page)
Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient. 

I think the problem might be that this term is called "electron transfer" rather than "electron transporter" one is much broader than the others.

Also, some of the xreferences to this term appear to be incorrect (and probably explain why such a huge blacklist is required) @pgaudet

**Comment by pgaudet** [comment]
We had indeed reviewed this part of the ontology with @jimhu-tamu and @dsiegele, and made the distinction between GO:0009055 electron transfer activity and general oxidoreductases. 
It's true that in papers 'electron carrier' is used as a synonym for oxidoreductase and for what GO calls electron 'transferers'. 

As far as cross-references are concerned, there are Reactome mappings that indeed need review: 
@deustp01 

    R-HSA-1362408 
    R-HSA-169260 
    R-HSA-2564826


Thanks, Pascale

**Comment by srengel** [comment]
thanks @ValWood and @pgaudet , i changed it to GO:0016730 oxidoreductase activity, acting on iron-sulfur proteins as donors.

**Comment by srengel** [comment]
oops reopening, because @pgaudet assigned @deustp01 .

**Comment by dsiegele** [comment]
@srengel @pgaudet @deustp01 @ValWood 

The definition for GO:0016730 oxidoreductase activity, acting on iron-sulfur proteins as donors is: Catalysis of an oxidation-reduction (redox) reaction in which an iron-sulfur protein acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor. 

But, in the reaction described in `https://www.ncbi.nlm.nih.gov/pubmed/24422557', the iron-sulfur protein (Dph1/Dph2) is the electron acceptor and KTI11 in the electron donor. 

Is GO:0016730 is meant to be used for reactions where the iron-sulfur protein is either the donor or the acceptor? If so, the term name and definition seem too specific. If not, is a new term needed, such as oxidoreductase activity, acting on iron-sulfur proteins as acceptors?

**Comment by pgaudet** [comment]
@dsiegele @srengel 

I am not much of a biochemist, but the overall reaction shown seems to be a transferase rather than an oxidoreductase (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3985478/figure/sch1/)

![image](https://user-images.githubusercontent.com/4782928/61205035-4239f580-a6ef-11e9-9292-72a723102197.png)

KTI11 seems to contribute to the reaction rather as some sort of chaperone than by an actual catalitic reaction, see https://www.nature.com/articles/nature09138?draft=journal&proof=true

> The three cysteine residues required to bind the cluster are conserved in Dph1 and two of the cysteine residues are conserved in Dph2. Thus, the heterodimer of Dph1–Dph2 should at least bind one [4Fe–4S] cluster and may be sufficient to catalyse the first step in vitro. Dph2, which has only two of the conserved cysteine residues, could either have a different catalytic function fr
om Dph1 or be regulatory. In vivo, Dph3 and Dph4 are also required for diphthamide biosynthesis3. These gene products may be required to keep the [4Fe–4S] cluster in a reduced state. This hypothesis is supported by the observation that Dph3 can bind iron and is redox active36. Alternatively, Dph3 and Dph4 may be required for proper assembly of the [4Fe–4S] clusters. The iron–sulphur-cluster assembly pathways in bacteria and the mitochondria of eukaryotes are known to involve J-domain-containing co-chaperone proteins, such as bacterial HscB and yeast Jac137,38, that are similar to Dph4. Confirmation of these functional assignments awaits detailed biochemical and structural studies.

I would be inclined to remove any catalytic reaction for KTI11. It could be annotated to 'iron binding' and 'contributes to' '2-(3-amino-3-carboxypropyl)histidine synthase. ' (see https://enzyme.expasy.org/EC/2.5.1.108). 

Input from @amorgat and @kaxelsen would be useful. 

Thanks, Pascale

**Comment by ValWood** [comment]
I would not annotate a MF for dph3 and dph4, it is really unclear what they do. I agree,  more likely to be a transferase. They are a type of zinc finger and this is the only catalytic activity I have seen associated with zinc fingers (for example ubiquitin ligase are zinc fingers). Oxidoreductase seems unlikely?

**Comment by kaxelsen** [comment]
@pascale. What your model does not show is that a DPH1/DPH2 dimer is capable of catalysing the transferase reaction (EC 2.5.1.108) on their own. As DPH2 uses the radical S-adenosyl-Lmethionine (radical SAM) mechanism it means a radical is formed on the enzyme itself which is used to break the SAM molecule and in this case form a 3-amino-3-carboxypropyl radical. In doing so DPH2 sacrifices a [4Fe-4S] cluster. To regenerate the [4Fe-4S] cluster to its reduced and active state, electrons are provided by DPH3 (Kti11). This protein again gets electrons from CBR1 as described here:
https://www.ncbi.nlm.nih.gov/pubmed/27694803

So in conclusion, DPH3 is not a transferase, it is an electron carrier but it is not involved in the electron transport chain that results in a transmembrane electron gradient.

I do not have an opinion about your definitions, but CBR1 -> DPH3 -> DPH2 could be considered being an electron transport chain?

**Comment by ValWood** [comment]
In GO 

GO:0022900    electron transport chain
is defined

Definition (GO:0022900 GONUTS page)
A process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient.

and 
GO:0009055    electron transfer activity
Molecular Function

Definition (GO:0009055 GONUTS page)
Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient.


So I think we wanted to make these specific for the process of  generation of an electrochemical gradient, rather than just to represent  electron "relocation" between chemical entities (which  happens in many  physiological contexts).

It seem that in this case, the term names need to be refined to match the definitions?

**Comment by pgaudet** [comment]
although... if we change the term label to 'electrochemical gradient-generating electron transport chain' - will users understand what we mean ? 

Thinking about it, I'd rather make a comment than changing the term label. 

Pascale

**Comment by dsiegele** [comment]
I agree with Pascale: It would be better to add a comment rather than
changing the term label.
